sites/partners/src/applications/PaperApplicationDetails/sections/DetailsHouseholdMembers.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext, useMemo } from "react"
+import { t, GridSection, MinimalTable, Button } from "@bloom-housing/ui-components"
+import { DetailsApplicationContext } from "../DetailsApplicationContext"
+import { MembersDrawer } from "../DetailsMemberDrawer"
+
+type DetailsHouseholdMembersProps = {
+  setMembersDrawer: (member: MembersDrawer) => void
+}
+
+const DetailsHouseholdMembers = ({ setMembersDrawer }: DetailsHouseholdMembersProps) => {
+  const application = useContext(DetailsApplicationContext)
+
+  const householdMembersHeaders = {
+    name: t("t.name"),
+    relationship: t("t.relationship"),
+    birth: t("application.household.member.dateOfBirth"),
+    sameResidence: t("application.add.sameResidence"),
+    workInRegion: t("application.details.workInRegion"),
+    action: "",
+  }
+
+  const householdMembersData = useMemo(() => {
+    const checkAvailablility = (property) => {
+      if (property === null) {
+        return t("t.n/a")
+      } else if (property) {
+        return t("t.yes")
+      }
+
+      return t("t.no")
+    }
+    return application?.householdMembers?.map((item) => ({
+      name: `${item.firstName} ${item.middleName} ${item.lastName}`,
+      relationship: item.relationship
+        ? t(`application.form.options.relationship.${item.relationship}`)
+        : t("t.n/a"),
+      birth:
+        item.birthMonth && item.birthDay && item.birthYear
+          ? `${item.birthMonth}/${item.birthDay}/${item.birthYear}`
```

Comment:
Same comment here about using moment formatting. What do you think?

Reply:
As we discussed on Slack, it's not necessary in this case.